fobnail-attester.c

```diff
@@ -0,0 +1,138 @@
+/**
+ * Some LICENSE
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <arpa/inet.h>
+#include <coap2/coap.h>
+#include <signal.h>
+#include <stdbool.h>
+
+static bool quit = false;
+static const char LISTEN_ADDRESS[] = "0.0.0.0";
+static unsigned int port = COAP_DEFAULT_PORT; /* default port 5683 */
+
+static void handle_sigint(int signum)
+{
+	quit = true;
+}
+
+static void coap_attest_handler(struct coap_context_t* ctx, struct coap_resource_t* resource,
+				struct coap_session_t* session,	struct coap_pdu_t* in,
+				struct coap_binary_t* token, struct coap_string_t* query,
+				struct coap_pdu_t* out)
+{
+	printf("Received message.\n");
```

Comment:
Please dump contents of the received message so that we can see what we receive from client

Reply:
> Please dump contents of the received message so that we can see what we receive from client.

you mean save in file?

Reply:
@0xDen print to terminal